src/main/java/io/vertx/ext/dropwizard/impl/VertxMetricsImpl.java

```diff
@@ -156,30 +158,9 @@ public DatagramSocketMetrics createMetrics(DatagramSocket socket, DatagramSocket
   }
 
   @Override
-  public <P> PoolMetrics<?> createMetrics(P pool, String name, int size) {
-    // Todo
-    return new PoolMetrics<Object>() {
-      @Override
-      public Object taskSubmitted() {
-        return null;
-      }
-      @Override
-      public void taskRejected(Object task) {
-      }
-      @Override
-      public void taskBegin(Object task) {
-      }
-      @Override
-      public void taskEnd(Object task, boolean succeeded) {
-      }
-      @Override
-      public boolean isEnabled() {
-        return false;
-      }
-      @Override
-      public void close() {
-      }
-    };
+  public <P> PoolMetrics<?> createMetrics(P pool, String poolType, String poolName, int maxPoolSize) {
```

Comment:
OK
Le 13 mai 2016 3:22 PM, "Julien Viet" <notifications@github.com> a écrit :

> In src/main/java/io/vertx/ext/dropwizard/impl/VertxMetricsImpl.java
> <https://github.com/vert-x3/vertx-dropwizard-metrics/pull/28#discussion_r63180892>
> :
>
> > -      }
> > -      @Override
> > -      public void taskBegin(Object task) {
> > -      }
> > -      @Override
> > -      public void taskEnd(Object task, boolean succeeded) {
> > -      }
> > -      @Override
> > -      public boolean isEnabled() {
> > -        return false;
> > -      }
> > -      @Override
> > -      public void close() {
> > -      }
> > -    };
> > +  public <P> PoolMetrics<?> createMetrics(P pool, String poolType, String poolName, int maxPoolSize) {
>
> no I'm holding this on my machine, but I will push at the same time than
> this PR
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/vert-x3/vertx-dropwizard-metrics/pull/28/files/91dadc4c6d1fed78f6f3ed4b35d3ef747cbfb4ff#r63180892>
>